### PR TITLE
INFRA-2408 - Move plugin-site-api to plugin-site-api image

### PR DESCRIPTION
**Pull request by halkeye**
Ready for deploying to new plugin-site-api image

Blocked by https://issues.jenkins-ci.org/browse/INFRA-2408

### CODE REVIEW

**Comment by halkeye** [comment]
There's no way this will actually work till a new repo is created :)

**Comment by halkeye** [comment]
[2020-01-03T22:13:02.179Z] [ERROR] Failed to execute goal org.apache.maven.plugins:maven-war-plugin:2.6:war (default-war) on project plugin-site-api: Execution default-war of goal org.apache.maven.plugins:maven-war-plugin:2.6:war failed: Plugin org.apache.maven.plugins:maven-war-plugin:2.6 or one of its dependencies could not be resolved: Failed to collect dependencies at org.apache.maven.plugins:maven-war-plugin:jar:2.6 -> org.apache.maven:maven-archiver:jar:2.6: Failed to read artifact descriptor for org.apache.maven:maven-archiver:jar:2.6: Could not transfer artifact org.apache.maven:maven-archiver:pom:2.6 from/to central (https://repo.maven.apache.org/maven2): Transfer failed for https://repo.maven.apache.org/maven2/org/apache/maven/maven-archiver/2.6/maven-archiver-2.6.pom: Connection timed out (Read failed) -> [Help 1]


I don't know why its timing out repeatedly

**Comment by timja** [comment]
It doesn’t look like it’s using our mirror, probably an issue upstream, we normally use something like infra.runMaven in the jenkinsfile so it handles that

File: Jenkinsfile [hunk 1]
```diff
@@ -51,7 +44,14 @@ node('docker&&linux') {
                     withEnv([
                         'DATA_FILE_URL=http://nginx/plugins.json.gzip',
                     ]) {
-                        sh 'mvn -B -Dmaven.test.failure.ignore verify'
+                        List<String> mvnOptions = ['-Dmaven.test.failure.ignore','verify']
+                        infra.runMaven(
```
Comment (timja) [inline Jenkinsfile]: end result was: 
`mvn --batch-mode --show-version --errors -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn -s /home/jenkins/workspace/Infra_plugin-site-api_PR-77@tmp/settings-azure.xml -Dmaven.test.failure.ignore verify`

perfect great improvement

**Pull request merged**